code/game/gamemodes/cult/runes.dm

```diff
@@ -874,18 +890,16 @@ var/list/sacrificed = list()
 		if (cultist == user) //just to be sure.
 			return
 		if(cultist.buckled || cultist.handcuffed || (!isturf(cultist.loc) && !istype(cultist.loc, /obj/structure/closet)))
-			user << "<span class='warning'>You cannot summon \the [cultist], for \his shackles of blood are strong.</span>"
+			user << "<span class='warning'>You cannot summon \the [cultist], for \his shackles are too strong.</span>"
 			return fizzle(user)
 		cultist.loc = src.loc
 		cultist.lying = 1
 		cultist.regenerate_icons()
 
-		var/dam = round(25 / (users.len/2))	//More people around the rune less damage everyone takes. Minimum is 3 cultists
-
 		for(var/mob/living/carbon/human/C in users)
 			if(iscultist(C) && !C.stat)
 				C.say("N'ath reth sh'yro eth d[pick("'","`")]rekkathnor!")
-				C.take_overall_damage(dam, 0)
+				user.rune_apply_damage(25, 30)

```

Comment:
See my previous point. This will remove a limb or remove a stupid amount of blood. _An inconsistent amount of blood/damage._ For no benefit.

Move it out of the loop and make the amount static.